webapp/components/setting_item_max.jsx

```diff
@@ -89,6 +89,7 @@ export default class SettingItemMax extends React.Component {
                             {clientError}
                             {submit}
                             <a
+                                id={this.props.title}
```

Comment:
I think this will clash with the save button

should be 
`id={this.props.title + 'Cancel'}`